docs/FAQ/gui-faq-en.md

```diff
@@ -20,147 +20,148 @@ Zonemaster
 
 <a name="q1"></a>
 #### 1. What is Zonemaster?
-Zonemaster is a program that was designed to help people check, measure and
-hopefully also understand how DNS (Domain Name System) works.
+Zonemaster is a program designed to help people check, measure and
+hopefully also understand how the DNS (Domain Name System) works.
 
-It consists of four basic parts:
+It consists of four components:
 
   1. Engine - a test framework that supports all functionality to perform DNS tests.
-  2. The CLI interface to the Engine.
-  3. Backend, a server that allows you to run Zonemaster tests and save results using
+  2. CLI - a command-line interface to the Engine.
+  3. Backend - a server that allows you to run Zonemaster tests and save results using
      a JSON-RPC API and a database.
-  4. GUI - a web interface to the backend.
+  4. GUI - a web interface to the Backend.
 
-When a domain (such as "zonemaster.net") is submitted to Zonemaster (CLI or
-GUI) it will investigate the domain’s general health by traversing the DNS from root
-(.) via the TLD (Top Level Domain, like .net) to the nameserver(s) that hosts
-the information about the specific domain (zonemaster.net). The different sanity checks
-conducted by the Zonemaster tool is documented in the [Test Requirements
-document](https://github.com/zonemaster/zonemaster/blob/master/docs/requirements/TestRequirements.md).
+When a domain (such as 'zonemaster.net') is submitted to Zonemaster (CLI or
+GUI) it will verify the domain name’s general health with a series of tests.
+The different sanity checks and corresponding tests conducted by Zonemaster can be
+found in the [Test Requirements] document.
 
 <a name="q2"></a>
 #### 2. Who is behind Zonemaster?
-Zonemaster is a joint project between [Afnic](https://www.afnic.fr/en/)
-(registry operator of .fr TLD and several other
-TLDs, e.g. .re, .pm, .tf, .wf, .yt and .paris) and
-[The Swedish Internet Foundation](https://internetstiftelsen.se/en/)
-(registry
-operator of .se and .nu TLDs).
+Zonemaster is a joint project between [AFNIC] (registry of '.fr' TLD and several other
+TLDs, e.g. '.re', '.pm', '.tf', '.wf', '.yt' and '.paris') and the [Swedish Internet Foundation]
+(registry of '.se' and '.nu' TLDs).
 
 <a name="q3"></a>
 #### 3. How can Zonemaster help me?
 The Zonemaster tool is oriented towards two user categories:
 
   - Users who are knowledgable about the DNS protocol.
-  - Users who just want to know whether the the domains owned or used by them
+  - Users who just want to know whether the domain name(s) they own or use
```

Comment:
I running text it is better to avoid "name(s)". In this text it does not matter if it says "the domain name they own" or "the domain names they own". The message will be the same.